src/tasks/dump.ts

```diff
@@ -504,18 +504,19 @@ async function createAllowances(
   { gasEstimator, requiredConfirmations }: CreateAllowancesOptions,
 ) {
   let lastTransaction: ContractTransaction | undefined = undefined;
+  let current_nonce = await signer.getTransactionCount();
+  const fee = await gasEstimator.txGasPrice();
```

Comment:
One thing we should consider is increasing the base fee multiplier in these cases (instead of the default 2)? My concern is that since the time to mine the transaction goes up, it could get "stuck" if the base fee increases too much?

Reply:
Do you think the way it was done before (computing the fee on each pass of the loop) would make things better? Depending on the node used, a single run of the loop still takes about 1s (roundtrip to get the signed transaction acknowledged).

Reply:
Having a resend mechanism would be one of the most important features to make the script work in an unattended cronjob.

Reply:
> Ok, then I will move the fee estimate back inside the loop.

I think this only mattered before because we were waiting for transactions to mine before firing the new one.